pyproject.toml

```diff
@@ -43,6 +43,10 @@ pyccel-test = "pyccel.commands.pyccel_test:pyccel_test_command"
 
 [project.urls]
 Homepage = "https://github.com/pyccel/pyccel"
+Documentation = "https://pyccel.github.io/pyccel/"
+Source = "https://github.com/pyccel/pyccel"
+Tracker = "https://github.com/pyccel/pyccel/issues"
+"Release notes" = "https://github.com/pyccel/pyccel/blob/devel/CHANGELOG.md"
```

Comment:
We will have to update this link when we make a new release... I suppose that could be done automatically? Maybe we just need to have a "release checklist" to remind us of these small things.

Reply:
Do we need to? On the devel branch new developments are explicitly listed in a "Unreleased" section so this should still be a valid link for a release

Reply:
Otherwise it should be possible to do something with an install hook: https://hatch.pypa.io/latest/how-to/config/dynamic-metadata/#implement-hook

to have a dynamic value that depends on the version (as the tag)

Reply:
Alternatively it seems we can link to the CHANGELOG which probably makes more sense as we don't have a compact Release Notes anywhere other than linkedin
![image](https://github.com/user-attachments/assets/f51ee58d-60a6-4555-b124-1a62044c5ad3)

Reply:
Done [8a92e8f](https://github.com/pyccel/pyccel/pull/2374/commits/8a92e8fd40b5bc01bd2344af401ad2d56d344742)

Reply:
The link I was suggesting gives information for all the releases. One just needs to click on a previous version once they get there! And it also allows them to diff between releases...

Anyway, I also like your last solution which is probably the simplest.